hugo/content/docs/use-exisitng-data/retrofit-a-data-explorer/doc.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Retrofit data explorers"
 weight: 1
-draft: false
+draft: true
```

Comment:
If we don't have the content, how about we remove this file altogether?

Reply:
Fixed